.github/workflows/main.yml

```diff
@@ -0,0 +1,91 @@
+name: GameCI 🎮
+
+on: [pull_request]
+
+jobs:
+  test:
+    name: Run Unit Tests 🧪
+    runs-on: ubuntu-latest
+    
+    steps:
+      # Checkout
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          lfs: false
+
+      # Cache
+      - uses: actions/cache@v3
+        with:
+          path: Library
+          key: Library-${{ hashFiles('Assets/**', 'Packages/**', 'ProjectSettings/**') }}
+          restore-keys: |
+            Library-
+          
+      # Apply Pre Processor Symbols (Compiler options) 
+      - name: Apply csc.rsp (pre-processor symbols)
+        run: echo -define:MY_SYMBOL >> sample-app/Assets/csc.rsp
+
+      # Test
+      - name: Run tests
+        uses: game-ci/unity-test-runner@v4
+        env:
+          UNITY_SERIAL: ${{ secrets.UNITY_SERIAL }}
+          UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
+          UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
+        with:
+          projectPath: sample-app
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+
+  buildPlatforms:
+    name: Build for ${{ matrix.targetPlatform }} with Unity Version ${{ matrix.unityVersion }}✨
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        projectPath:
+          - sample-app
+        targetPlatform:
+          - Android # Build an Android player.
+          - iOS # Build an iOS player.
+        unityVersion:
+          - 6000.0.26f1
+          - auto
+        #excluding Android auto because building to APK breaks because of older Unity Versions
```

Comment:
This note indicates that the CI is doing exactly what it was intended to do and catching build errors on older versions.

What exactly is breaking on Android and what options do you think we have to fix it?